client/stylesheets/_toCat.scss

```diff
@@ -1156,6 +1156,7 @@ HELP | ABOUT | FAQ | SAFETY | PRIVACY | CONTACT **/
     height: 80px !important;
     border-radius: 4px !important;
     border: 1px solid #d3d3d3 !important;
+    vertical-align: middle !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces